coreneuron/nrniv/cellorder1.cpp

```diff
@@ -290,18 +288,18 @@ static void ident_statistic(VecTNode& nodevec, size_t ncell) {
     size_t maxlevel = level_from_root(nodevec);
 
     // # in each level
-    vector<vector<size_t> > n_in_level(maxlevel + 1);
-    for (size_t i = 0; i <= maxlevel; ++i) {
-        n_in_level[i].resize(ncell / groupsize);
+    std::vector<std::vector<size_t> > n_in_level(maxlevel + 1);
```

Comment:
I think this `> >` instead of `>>` was coming from custom clang-format setting in the coreneuron project (which will be replaced based on hpc-coding-convention.